.github/workflows/python-test.yml

```diff
@@ -40,7 +40,6 @@ jobs:
       postgres:
         image: postgres
         env:
-          POSTGRES_DB: testdb

```

Comment:
Why are we removing this exactly and what's the consequence?

Reply:
@jsvgoncalves this resolves the unit tests assuming a fresh install where we haven't created the database yet.

Reply:
Still not clear what the consequence of removing the DB name from the env vars is.

Reply:
@jsvgoncalves the postgresql server is created before running the unit tests but the `testdb` database is not. The unit tests then test multiple scenarios including the fresh install where there is no created database.